arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/raise/RaiseAccumulate.kt

```diff
@@ -502,6 +502,9 @@ public open class RaiseAccumulate<Error>(
   override fun <A> Iterable<Either<Error, A>>.bindAll(): List<A> =
     mapOrAccumulate { it.bind() }
 
+  override fun <A> NonEmptyList<Either<Error, A>>.bindAll(): NonEmptyList<A> =
+    requireNotNull(mapOrAccumulate { it.bind() }.toNonEmptyListOrNull())
```

Comment:
Which `mapOrAccumulate` is this resolving too if we have to wrap it in `requireNotNull`

Reply:
It uses the `mapOrAccumulate` defined for `Iterable` located in [the line](https://github.com/arrow-kt/arrow/pull/2995/files/75564ed5fa19db1f9b7a92b9541cc826dafe1c62#diff-faa31a2c3b82d4997f537af6af5e04db3cfa2fdd591098b85bf1bb6deb9f21b6R497). I could also change the implementation to:
```
requireNotNull(raise.mapOrAccumulate(this) { it.bind() }.toNonEmptyListOrNull())
```

Reply:
Ah, I thought there was also one defined for `NonEmptyList`. My bad.